.gitignore

```diff
@@ -1,7 +1,9 @@
 # exclude everything
 arm_diags/model/*
 arm_diags/observation/*
-arm_diags/cmip/*
+arm_diags/testmodel/*
+arm_diags/cmip5/*
+arm_diags/cmip6/*
 arm_diags/build/*
 arm_diags/dist/*
 
```

Comment:
folders: `egg-info` and `__pychache__` may need to be included in .gitignore as well, so that they don't get pushed to remote.

Reply:
I have updated the .gitignore. However, I cannot push it due to the error described in the attachment.

![Screenshot from 2022-09-19 12-49-04](https://user-images.githubusercontent.com/60485850/191103925-d2d7bcd4-e5d7-404c-8cf6-cdee06340917.png)

Reply:
Hey Xiaojian, it looks like `build/lib` and `dist` need to be excluded as well. I will try to see if I can fix the conflicts you see. 

A minor question:
The script: annual_cycle_aci.py and annual_cycle.py are pretty similar, I'm wondering if it is possible to reuse modules from one of both? Or they have some fundamental difference?

Reply:
Hi Jill, I update the gitignore successfully by doing 'git pull' first, and the pushing is done. But should I also remove those folder-should-be-ignored on the current existing ARM-Diags-V3 remote branch?

And yes, there is a fundamental difference in the ACI input data. Due to the potential diurnal cycles in the aerosol data, it will not be representative if the annual cycle is calculated from input that follows the structure of the original input (which is in monthly mean format). So the annual_cycle_aci.py is written specifically to accommodate the ACI input data (which is in climatology format).

Reply:
Good to know! Yes, please put files that needs to be ingored in the gitigmore and remove those excessive files. And push again.

Reply:
Thanks for the clarification on the specially treated ACI data structure. Regarding to model input , it seems like the expected input is also climatology annual cycle?

Reply:
Hi Jill, I have managed to remove the unnecessary files on the branch and pushed the new one on it. 
As for the model input, that would be the same as the original monthly data structure, so that the modeler do not have to create another particular file. And also is because the model outputs aerosols in a much more time-continuing way than the observation.

Reply:
Thanks for clarifying!